packages/documentation/src/stories/foundation/icons/icons.stories.mdx

```diff
@@ -0,0 +1,288 @@
+import { Meta, Canvas, Story } from '@storybook/addon-docs';
+import { Search } from './icons.components';
+
+<Meta
+  title="Foundations/Icons"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+    viewMode: 'docs',
+  }}
+/>
+
+# Icons
+
+<p className="lead">
+  The official Swiss Post Icon Library for the web with over 800 high-quality scalable vector
+  graphics.
+</p>
+
+The icons are designed to work with the Swiss Post Design System components. They are SVGs, so they scale and can be styled with CSS.
+
+<!-- this is an invisible, navigation-only story -->
+
+<Story name="Start">
+  <div />
+</Story>
```

Comment:
Remove this, if we extract the other stories to a post-icon component documentation

Reply:
Moved this to a separate issue #1083